### PR TITLE
Update issue templates

### LINKED ISSUE #9058: Separate status-react issue templates into bug & feature
@jakubgs would you do the honor? Assuming we can override the existing templates in status-react (`Full Bug Report` and `Simple Issue Report`), I think this makes more sense:

### Full Bug Report --> Feature Issue

[Old template in hackmd](https://hackmd.io/wR7LASy4R2uT2uy0XK2DXg)
[Proposed template in hackmd](https://hackmd.io/8Qnps_oQRWWcti-7kMcYxw)

### Simple Issue Report --> Bug Report

[Old template in hackmd](https://hackmd.io/OMVZwJ8yS9KUdeYHlS7g3A)
[Proposed template in hackmd](https://hackmd.io/z_83ZYUtTZC2fjhoolTFsw)

### PR DESCRIPTION
**Pull request by jakubgs**
Resolves #9058

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-issue.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+### User Story
+
+As a <user|developer|...>, I want to <task> **so that** <goal>. 
+
+### Description
+
+[comment]: # (Describe the feature you would like and include designs and Figma links if available.)
+*Summary*: ...
+
```
Comment (hesterbruikman) [inline .github/ISSUE_TEMPLATE/feature-issue.md:23]: @jakubgs can you please add:

### Uses cases
[comment]: # Describe the use cases the implementation should cover. E.g. New users, existing users, public chat, group chat, etc

Comment (jakubgs) [reply .github/ISSUE_TEMPLATE/feature-issue.md:23]: Done.

**Review by rachelhamlin — APPROVED** [review]

**Comment by yenda** [comment]
I think we still need a simple issue kind of template for things like tech debt for instance which aren't really bugs nor features?

**Comment by rachelhamlin** [comment]
Sure! That’s fair. Do you have the permission to create one?

**Comment by jakubgs** [comment]
@rachelhamlin @yenda just post the contents here or in https://notes.status.im/ and I'll add it.
Or just push to this branch if you have something to add.

**Comment by yenda** [comment]
@jakubgs we could just revert the deletion of simple issue, maybe leaving only the #Problem section

**Comment by jakubgs** [comment]
Done.

**Comment by yenda** [comment]
@PombeirP @flexsurfer @cammellos @siphiuel all OK to merge?

**Comment by vitvly** [comment]
I'm good.

**Pull request merged**